### PR TITLE
Switching tabs: minor optimization

### PR DESCRIPTION
**Pull request by ivandrofly**

### CODE REVIEW

File: src/Forms/Main.cs [hunk 1]
```diff
@@ -6087,29 +6087,29 @@ private void ReloadFromSourceView()
...
 
                     int index = 0;
-                    foreach (object obj in comboBoxSubtitleFormats.Items)
+                    foreach (string formatName in comboBoxSubtitleFormats.Items)
                     {
-                        if (obj.ToString() == format.FriendlyName)
+                        if (formatName == format.FriendlyName)
                         {
                             comboBoxSubtitleFormats.SelectedIndex = index;
+                            break;
```
Comment (ivandrofly) [inline src/Forms/Main.cs]: break as soon as the format is found

File: src/Forms/Main.cs [hunk 2]
```diff
@@ -6087,29 +6087,29 @@ private void ReloadFromSourceView()
                 if (!string.IsNullOrWhiteSpace(textBoxSource.Text))
                 {
                     SubtitleFormat format = GetCurrentSubtitleFormat();
-                    var list = textBoxSource.Lines.ToList();
-                    format = new Subtitle().ReloadLoadSubtitle(list, null, format);
+                    var lines = new List<string>(textBoxSource.Lines);
+                    format = new Subtitle().ReloadLoadSubtitle(lines, null, format);
```
Comment (ivandrofly) [inline src/Forms/Main.cs]: 2n => n

File: src/Forms/Main.cs [hunk 3]
```diff
@@ -22680,7 +22680,7 @@ private void TabControlSubtitleSelecting(object sender, TabControlCancelEventArg
                     return;
                 }
 
-                SubtitleFormat format = new Subtitle().ReloadLoadSubtitle(textBoxSource.Lines.ToList(), _fileName, currentFormat);
+                SubtitleFormat format = new Subtitle().ReloadLoadSubtitle(new List<string>(textBoxSource.Lines), _fileName, currentFormat);
```
Comment (ivandrofly) [inline src/Forms/Main.cs]: 2n => n

File: src/Forms/Main.cs (hunk 2 above)
Comment (niksedk) [reply src/Forms/Main.cs]: Ran some tests (1000 iterations in release mode without VS) and `textBoxSource.Lines.ToList();` and `new List<string>(textBoxSource.Lines)` are *very* close - ToList seems a little faster with large subtitles and new List seems a little faster with small subtitles, so not really a clear winner IMO.

File: src/Forms/Main.cs (hunk 1 above)
Comment (niksedk) [reply src/Forms/Main.cs]: :)

**Pull request closed without merging**